src/nginxconfig/generators/conf/website.conf.js

```diff
@@ -287,12 +289,21 @@ export default (domain, domains, global) => {
         serverConfig.push(['# handle .php', '']);
 
         const loc = `location ~ ${domain.routing.legacyPhpRouting.computed ? '[^/]\\.php(/|$)' : '\\.php$'}`;
+
+        const fastcgiPass = {
+            fastcgi_pass: domains.some(d => d.php.php.computed) && domain.php.phpBackupServer.computed !== ''
+                ? phpUpstream(domain) : phpPath(domain),
+        };
+
         if (global.tools.modularizedStructure.computed || domain.php.wordPressRules.computed) {
             // Modularized
-            serverConfig.push([loc, { include: 'nginxconfig.io/php_fastcgi.conf' }]);
+            serverConfig.push([loc, {
+                include: 'nginxconfig.io/php_fastcgi.conf',
+                ...fastcgiPass,
+            }]);
         } else {
             // Unified
-            serverConfig.push([loc, phpConf(domains, global)]);
+            serverConfig.push([loc, phpConf(domains)]);
```

Comment:
```suggestion
            serverConfig.push([loc, {
                ...phpConf(domains),
                ...fastcgiPass,
            }]);
```
This needs to be updated to also have the `fastcgi_pass` set.